game/scripts/vscripts/items/siege_mode.lua

```diff
@@ -58,6 +73,31 @@ end
 
 --------------------------------------------------------------------------------
 
+function modifier_item_siege_mode_siege:OnCreated( event )
+	local spell = self:GetAbility()
+
+	spell.mod = self
+
+	self.atkRange = spell:GetSpecialValueFor( "siege_attack_range" )
```

Comment:
Actually, in this case, there is no reason to do this, as the only way to apply the modifier ( aka activating Siege Mode ) _will_ remove the modifier should it already exist. So it can't actually be refreshed.